plasmapy/physics/parameters.py

```diff
@@ -1425,12 +1307,9 @@ def upper_hybrid_frequency(B, n_e):
 
     """
 
-    try:
-        omega_pe = plasma_frequency(n=n_e)
-        omega_ce = gyrofrequency(B)
-        omega_uh = (np.sqrt(omega_pe**2 + omega_ce**2)).to(u.rad / u.s)
-    except Exception:
-        raise ValueError("Unable to find upper hybrid frequency.")
+    omega_pe = plasma_frequency(n=n_e)
+    omega_ce = gyrofrequency(B)
+    omega_uh = (np.sqrt(omega_pe**2 + omega_ce**2)).to(u.rad / u.s)
```

Comment:
move unit conversion to return line

Reply:
That's not even a standard I was aware we had! :D

Reply:
It just sort of appeared in some functions, but not others. Personally I like it since you can skip to the end of the function to take a glance at the units.

Reply:
Might be worth documenting as a standard :smile:

Reply:
I kinda like it, but I think function annotations (`-> u.m/u.s`) may be an even better way to accomplish that! I've been meaning to go through our code and use `@u.quantity_input` wherever we can, as that takes advantage of it.

Reply:
So let's not make something I intend to enjoy refactoring a standard, okay? :D

Reply:
#314 ;)

Reply:
Sure, checking the input quantities should do this implicitly if the code/theory are correct, but doing the unit conversion at the end is a nice sanity check. We could of course just move this check into the relevant tests modules.

Reply:
What if I were to tell you Astropy's quantity input does both input and return value checking?

Reply:
https://godofall.files.wordpress.com/2014/06/mind-blown.gif